students/swietczak_monika/lesson_11_argparse_and_math/filling_the_gaps_argparse.py

```diff
@@ -0,0 +1,42 @@
+import os
+import re
+import argparse
+
+
+def check_args(args=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-p", "--prefix",
+                        help="prefix",
+                        default="spam")
+    parser.add_argument("-d", "--directory",
+                        help="directory",
+                        default=r"C:\Python36\gaps")
+    results = parser.parse_args(args)
+    return results.prefix, results.directory
```

Comment:
In real life it would be better if you left that unpacked i.e. `return results` This way you make it easier to extend that to use more parameters: in case other function would use that call you don't have to change that (accessing `results.prefix` will work regardless of the new parameters added). 

Also you avoid possible mistakes from swapping the order. It is general rule that it's better to use keyword parameters and also limit the positional parameters to maximum of three.